README.md

```diff
@@ -147,9 +147,17 @@ To setup the client to use your deployment, you'll need to pass in `OpenAIClient
 
 ```csharp
 var auth = new OpenAIAuthentication("sk-apiKey");
-var settings = new OpenAIClientSettings(resourceName: "your-resource", deploymentId: "your-deployment-id");
+var settings = new OpenAIClientSettings(resourceName: "your-resource", deploymentId: "your-deployment-id", apiVersion: "your-api-version");
 var api = new OpenAIClient(auth, settings);
 ```
+For example, when in the Azure Playground, you can click the `View Code` button and view a URL like this:
+
+https://myresource.openai.azure.com/openai/deployments/GPT-35-Turbo/chat/completions?api-version=2023-03-15-preview
+
+That would be translated as follows:
+`resourceName`: myresource
+`deploymentId`: GPT-35-Turbo
```

Comment:
Do the deployment ids really correspond to the available model?

Reply:
This is what I *just-got-working* on my project 😎

Reply:
Yeah *you* create that `deploymentId` when you create a deployment. This is a *very* Microsoft thing that doesn't exist on OpenAI